app/config/collections.php

```diff
@@ -819,75 +819,114 @@
                 'filters' => [],
             ],
             [
-                '$id' => ID::custom('schedule'),
+                '$id' => ID::custom('resourceId'),
                 'type' => Database::VAR_STRING,
                 'format' => '',
-                'size' => 100,
+                'size' => Database::LENGTH_KEY,
                 'signed' => true,
                 'required' => false,
                 'default' => null,
                 'array' => false,
                 'filters' => [],
             ],
             [
-                '$id' => ID::custom('active'),
-                'type' => Database::VAR_BOOLEAN,
-                'signed' => true,
-                'size' => 0,
+                '$id' => ID::custom('status'),
```

Comment:
Can "active" be as constant INT?

Reply:
We will refactor all this kind of ENUMs like fields as a separate PR.